# run a code and return a message when an error occur
try:
    n1 = int(input("Ingresa primer número: "))
except:
    print("ocurrió un error")
